ontology/yaml/resources/HVAC/entity_types/RP.yaml

```diff
@@ -37,4 +37,30 @@ RP_DPBHCC_FTC:
   implements:
   - RP
   - DPBHCC
-  - FTC
\ No newline at end of file
+  - FTC
+
+#Entity Types for DE-FRA-NEM32
+
+RP_CAM_DPHC2_ZTC:
```

Comment:
its unclear to me how these devices actually work. they control zone temperature by opening/closing two-position valves?